v2/fleet-local/proxy/proxy@.service

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Proxy { Nginx/HAProxy } @ %i
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=core
+Restart=always
+RestartSec=20
+TimeoutStartSec=0
+Environment="IMAGE=etcdctl get /images/proxy"  
+
+ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull $($IMAGE)"
+ExecStartPre=-/usr/bin/docker kill proxy
+ExecStartPre=-/usr/bin/docker rm proxy
+
+# NOTE: it's critical to source the etcdctl.sh file so that etcd connects to the correct cluster.
+ExecStart=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && \
+  docker run \
+    --name proxy \
+    --net='host' \
+    $($IMAGE) nginx -g \"daemon off;\""  #HOW DO WE MAKE THIS CONDITIONAL? etcd?
```

Comment:
I guess because this service is explicitly dependant on capcom... `/capcom/proxy`? We should definitely talk about a standard at some point